examples/application_commands/options.py

```diff
@@ -0,0 +1,26 @@
+from nextcord import SlashOption
+from nextcord.ext import commands
+from nextcord.interactions import Interaction
+
+bot = commands.Bot(command_prefix="/")
```

Comment:
why not just use client

Reply:
Not really the best practice imo

Reply:
It should be consistent with the other examples though

Reply:
@techpluscraft most of the examples use `commands.Bot` smh